jpa/spec-tests/src/main/java/ee/jakarta/tck/persistence/se/entityManagerFactory/Client1.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2007, 2023 Oracle and/or its affiliates. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package ee.jakarta.tck.persistence.se.entityManagerFactory;
```

Comment:
@gurunrao does `se`  in the package name mean?  I just wanted to make sure it isn't a typo.  Thanks!

Reply:
The package name is not a typo - original tests can be found at https://github.com/jakartaee/platform-tck/tree/master/src/com/sun/ts/tests/jpa/se